lib/vrf.c

```diff
@@ -712,6 +729,29 @@ DEFUN_NOSH (vrf,
 	return vrf_handler_create(vty, vrfname, NULL);
 }
 
+DEFUN_NOSH (vrf_name_set,
```

Comment:
Why `DEFUN_NOSH` if this command doesn't change the vty node?

Also, what happens if I use `vrf-default-name NAME` in zebra and not in the other daemons? Can't that lead to inconsistencies?